e2e/meta_test/test_schema_nodes_labels/test.yml

```diff
@@ -0,0 +1,8 @@
+query: >
+    CALL meta.schema() YIELD nodes, relationships UNWIND nodes AS node UNWIND node.labels AS label RETURN label;
```

Comment:
Also, for this part we don't have guarantee of order, maybe we can add here ORDER BY label DESC and then be sure.